EIPS/eip-3091.md

```diff
@@ -0,0 +1,45 @@
+---
+eip: 3091
+title: Block Explorer API Routes
+author: Pedro Gomes (@pedrouid)
+discussions-to: https://github.com/ethereum/EIPs/pull/3091
+status: Draft
+type: Standards Track
+category: Interface
+created: 2020-11-02
+---
+
+## Simple Summary
+This proposal brings standardization between block explorers API routes when linking transactions, blocks, accounts and tokens. 
+
+## Abstract
+Currently wallets will link transactions and accounts to block explorers web pages but as chain diversity and layer two solutions grow it becomes harder to maintain a consistent user experience. Standardizing the API routes to these links improves interoperability between wallets and block explorers.
+
+## Motivation
+Adding new chains or layer two solutions becomes harder given these endpoints are inconsistent. This EIP makes RPC endpoints like EIP-2015 and EIP-2016 more feasible.
+
+## Specification
+Block explorers will route their webpages accordingly for the following data:
+
+### Blocks
+<BLOCK_EXPORER_URL>/block/<BLOCK_HASH_OR_HEIGHT>
+
+
+### Transactions
+<BLOCK_EXPORER_URL>/tx/<TX_HASH>
+
+### Accounts
+<BLOCK_EXPORER_URL>/address/<ACCOUNT_ADDRESS>
+
+
+### ERC-20 Tokens
+<BLOCK_EXPORER_URL>/token/<TOKEN_ADDRESS>
+
+## References
+
+- Related EIPs
+  - [EIP-2015: Wallet Update Ethereum Chain JSON-RPC Method (`wallet_updateChain`)](./eip-2015.md)
+  - [EIP-2016: Wallet Add Ethereum Chain RPC Method (`wallet_addEthereumChain`)](./eip-2016.md)
+
```

Comment:
```suggestion
```
This isn't a standard EIP section and we prefer not to have any non-standard sections (so we can write tooling easier).  In this case, we can inline the URL for the one link to another EIP.